packages/@best/api-db/src/sql/sqlite/db.ts

```diff
@@ -5,18 +5,21 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
 
-import sqlite, { Database } from 'sqlite'
+import { open, Database } from 'sqlite'
+import sqlite3 from 'sqlite3'
 import { SQLDatabase, SQLQueryResult } from '../db'
 import { migrate } from './migrate';
 import { ApiDatabaseConfig } from '@best/types';
 
+sqlite3.verbose() // enable verbose mode
+
 export default class SQLiteDatabase extends SQLDatabase {
     dbPromise: Promise<Database>
     migrated = false;
 
     constructor(config: ApiDatabaseConfig) {
         super()
-        this.dbPromise = sqlite.open(config.uri, { verbose: true })
+        this.dbPromise = open({ filename: config.uri, driver: sqlite3.Database })
```

Comment:
See the new `open` API: https://github.com/kriasoft/node-sqlite#opening-the-database